src/connectors/geo-search/connectGeoSearch.js

```diff
@@ -332,6 +358,27 @@ const connectGeoSearch = (renderFn, unmountFn) => {
 
         return nextState;
       },
+
+      getWidgetState(uiState, { searchParameters }) {
+        const boundingBox = searchParameters.insideBoundingBox;
+        if (boundingBox) {
+          return {
+            ...uiState,
+            boundingBox,
+          };
+        }
+        return uiState;
+      },
+
+      getWidgetSearchParameters(searchParameters, { uiState }) {
+        if (uiState.boundingBox) {
+          return searchParameters.setQueryParameter(
+            'insideBoundingBox',
+            uiState.boundingBox
+          );
+        }
+        return searchParameters;
+      },
```

Comment:
We can add tests for those two functions.

Reply:
Of course!